ui/app/components/clients/chart-container.hbs

```diff
@@ -0,0 +1,54 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{! HBS display template for rendering title, description and stat boxes with a chart on the right }}
```

Comment:
ohh I like the description on the hbs file especially if this is a template only component.